src/js/hooks/useSessionStorage.js

```diff
@@ -0,0 +1,9 @@
+const useSessionStorage = (key, defaultValue) => {
+  const val = JSON.parse(sessionStorage.getItem(key)) ?? defaultValue;
+  const setValue = (newValue) => sessionStorage.setItem(key, JSON.stringify(newValue));
+  const clearValue = () => sessionStorage.removeItem(key);
+
+  return [val, setValue, clearValue];
```

Comment:
wouldn't it be better to return it as object, so we would be able to get rid of the confusing:
```js
const [cachedData,]
```
it would get even more weird if we wanted to use the first and the third property:
```js
const [cachedData, null, clearValue]
```

Reply:
I modeled this `useSessionStorage` hook after `useState` where we destruct the value and setter from an array and accessing nth value like `[,,clearFunction] = useSessionStorage()` is rather common and is aligned with react conventions.

I don't feel like one solution is better than the other, it's what we prefer, both are correct.

Reply:
ok, for me it just looks weird when we have a comma without anything after that, but if you feel like it's better, then I'm fine, it was not a change request